### DISCUSSION TITLE
Repo Name Changing

### DISCUSSION THREAD

**Discussion by ad3154** [Open-Agriculture/AgIsoStack-plus-plus#186] [Announcements]
Just a quick announcement, the repo's name will officially change from "ISO11783-CAN-Stack" to "Isobus-plus-plus" this weekend to reflect our project's branding.

This should not break access for anyone, and GitHub should properly redirect everything to the new repo name, but you might want to consider updating your local copies of this repo to use the new name for your upstream remote. This is not required, just a suggestion.

I'd also like to thank everyone for your continued support of this project - we're making constant improvements and exciting features like ISO11783-10 Task Controller Client support are in active development. Even though this is a labor of love produced on our nights and weekends, it is rapidly evolving into a high quality, comprehensive ISOBUS solution that is entirely free for everyone to use.

Thanks,

Adrian

**Comment by arcostasi** [comment]
Hello Adrian,

I'm thrilled to inform you that I was able to successfully compile the project on my Windows 10 64, and was able to test it with the PCAN-USB from Peak System. The entire process was seamless and hassle-free, and I was able to run the program without any issues.

I must say that the design and functionality of the project is top-notch. It is evident that a lot of thought and expertise went into its creation, and I am sure it will be a valuable resource for many users.

Thank you for all your hard work and for sharing this incredible project with the community. You have truly set the bar high for software development and I look forward to seeing what other exciting things you have in store for the future.

**Reply by ad3154** [reply]
Thank you for the kind words and your support! I am glad you were able to get started with this project without too much hassle 😁
